extensions/sql-migration/src/dialog/feedbackDialog.ts

```diff
@@ -0,0 +1,184 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as azdata from 'azdata';
+import * as vscode from 'vscode';
+import { IconPathHelper } from '../constants/iconPathHelper';
+import * as loc from '../constants/strings';
+import { sendSqlMigrationActionEvent, TelemetryActions, TelemetryViews } from '../telemtery';
+
+export class FeedbackDialog {
+
+	private static readonly DialogName: string = 'FeedbackDialog';
+
+	private _dialog!: azdata.window.Dialog;
+	private _buttonGroup!: azdata.FlexContainer;
+	private _isOpen: boolean = false;
+	private _feedbackRating?: number;
+	private _feedbackText?: string;
+
+	constructor() {
+	}
+
+	public async openDialog() {
+		if (!this._isOpen) {
+			this._isOpen = true;
+			this._dialog = azdata.window.createModelViewDialog(
+				'',
```

Comment:
Would suggest giving this dialog a name (non-localized) so we can identify it in telemetry